Compiler/Project.toml

```diff
@@ -1,3 +1,6 @@
 name = "Compiler"
 uuid = "807dbc54-b67e-4c79-8afb-eafe4df6f2e1"
 version = "0.0.2"
+
+[compat]
+julia = "1.10"
```

Comment:
It would be possible to install the fallback implementation of Compiler.jl on almost any Julia version, so there might not be a strict necessity to set the minimum compatible Julia version to 1.10. However, I’m not a fan of unnecessarily broad compatibility, and in my opinion, it’s sufficient to support at least the current LTS version.